.appveyor.yml

```diff
@@ -2,8 +2,10 @@ version: "{build} ~ {branch}"
 
 os: Visual Studio 2017
 
-# Do not build feature branch with open Pull Requests
-skip_branch_with_pr: true
+branches:
+  only:
+    - staging
+    - trying
```

Comment:
@Hywan could be possible to know why we only want to test only in `staging` or `trying` branches?

Reply:
That's what you're supposed to do: https://bors.tech/documentation/getting-started/

Reply:
Correct.